src/UploadedFile.php

```diff
@@ -16,6 +16,18 @@
 
 class UploadedFile implements UploadedFileInterface
 {
+    private static $errorMessages = [
```

Comment:
Make this a constant instead; PHP has supported constant array values since 5.6, and they're a better fit here than a static member.